appveyor.yml

```diff
@@ -11,12 +11,15 @@ environment:
 platform:
   - x64
 install:
-  - ps: .\appveyor.ps1
+  - ps: .\appveyor\hardwareinfo.ps1
```

Comment:
+1000000 on this, we could even move appveyor.yml in there, since that can be in a custom place (not for .travis or build.yaml though)